Makefile

```diff
@@ -0,0 +1,60 @@
+# Image URL to use all building/pushing image targets
+IMG ?= quay.io/nmagnezi/observatorium-operator:latest
```

Comment:
created a new quay repo for this https://quay.io/repository/observatorium/observatorium-operator?namespace=observatorium

Reply:
Fixed.
Still with previous image in the operator.yaml until we have an imaged pushed.